assets/models/organelles/Flagellum.glb.import

```diff
@@ -16,6 +16,7 @@ nodes/root_name="Scene Root"
 nodes/root_scale=1.0
 nodes/custom_script=""
 nodes/storage=0
+nodes/use_legacy_names=true
```

Comment:
I'm guessing this change doesn't break anything but in my experience it's good to avoid unrelated changes in a PR as much as possible.